src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -110,60 +99,69 @@ const ReleaseFootnotesSection = ({
-              onOpen={footnoteFormControls.create}
-              onCancel={footnoteFormControls.cancel}
-              onSubmit={footnoteFormControls.save}
-              isFirst={footnotes && footnotes.length === 0}
-              footnoteMeta={footnoteMeta}
-              footnoteMetaGetters={footnoteMetaGetters}
-            />
-          )}
-          {footnoteMeta && (
-            <>
-              <FootnotesList
-                footnotes={footnotes}
-                footnoteMeta={footnoteMeta}
-                footnoteMetaGetters={footnoteMetaGetters}
-                footnoteFormControls={footnoteFormControls}
-                canUpdateRelease={canUpdateRelease}
+      {footnotesData === undefined ||
+        (!footnotesData.hasSufficientData && (
+          <p>
+            Before footnotes can be created, relevant data files need to be
+            uploaded. That can be done in the{' '}
+            <Link to="#data-upload">Data uploads section</Link>.
+          </p>
+        ))}
+      {footnotesData && loading && <LoadingSpinner />}
+      {footnotesData &&
+        !loading &&
+        footnotesData &&
+        footnotesData.hasSufficientData &&
+        footnotesData.footnoteMeta &&
+        footnotesData.footnoteMetaGetters && (
+          <>
+            {footnotesData.canUpdateRelease && (
+              <FootnoteForm
+                {...footnoteForm}
+                footnote={undefined}
+                onOpen={footnoteFormControls.create}
+                onCancel={footnoteFormControls.cancel}
+                onSubmit={footnoteFormControls.save}
+                isFirst={
+                  footnotesData.footnotes &&
+                  footnotesData.footnotes.length === 0
+                }
+                footnoteMeta={footnotesData.footnoteMeta}
+                footnoteMetaGetters={footnotesData.footnoteMetaGetters}
               />
-              {typeof footnoteToBeDeleted !== 'undefined' && (
-                <ModalConfirm
-                  title="Confirm deletion of footnote"
-                  onExit={() => setFootnoteToBeDeleted(undefined)}
-                  onCancel={() => setFootnoteToBeDeleted(undefined)}
-                  onConfirm={() => {
-                    footnotesService
-                      .deleteFootnote((footnoteToBeDeleted as Footnote).id)
-                      .then(() => setFootnoteToBeDeleted(undefined))
-                      .then(getFootnoteData)
-                      .catch(handleApiErrors);
-                  }}
-                >
-                  The footnote:
-                  <p className="govuk-inset-text">
-                    {(footnoteToBeDeleted as Footnote).content}
-                  </p>
-                </ModalConfirm>
-              )}
-            </>
-          )}
-        </>
-      )}
+            )}
+            {footnotesData.footnoteMeta && (
+              <>
+                <FootnotesList
+                  footnotes={footnotesData.footnotes}
```

Comment:
You could just spread the `footnotesData` into this component's props to be more concise